src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random rand = new Random();
+
+    public Random getRand() {
+        return rand;
+    }
+
+    public void setRand(Random rand) {
+        this.rand = rand;
+    }
```

Comment:
The getter and setter for `rand` are not necessary and should be removed. If the `Random` instance is moved to a local variable within `getRandomBall`, there would be no need for these methods.